### PR TITLE
Onboarding: Remove skip button and add 'X' close button instead (closes #937)

### LINKED ISSUE #937: Onboarding Tweaks 02: Replace Skip button with “x” in the Welcome slide
- [x] remove Skip button
- [x] add x in the corner to skip it

screenshot 2017-11-06 16 06 44_preview

### PR DESCRIPTION
**Pull request by broskoski**

### CODE REVIEW

File: apps/onboarding/scenes/welcome/index.styl [hunk 1]
```diff
@@ -23,6 +23,20 @@
     font-size font-size-lg
     margin-top 2em
 
+    &__skip
+      position absolute
+      top 0
+      right 0
+      padding unit unit 40px 40px
```
Comment (dzucconi) [inline apps/onboarding/scenes/welcome/index.styl]: use ems

File: apps/onboarding/scenes/welcome/index.styl [hunk 2]
```diff
@@ -23,6 +23,20 @@
...
+      cursor pointer
+      opacity 0.45
+      z-index rotate_z + 1
```
Comment (dzucconi) [inline apps/onboarding/scenes/welcome/index.styl]: what is this

Comment (dzucconi) [reply apps/onboarding/scenes/welcome/index.styl]: also there's &times; the icon font x is chunky and ridiculous looking

Comment (broskoski) [reply apps/onboarding/scenes/welcome/index.styl]: copy pasta from modal style

Comment (dzucconi) [reply apps/onboarding/scenes/welcome/index.styl]: please don't do this to me

Comment (broskoski) [reply apps/onboarding/scenes/welcome/index.styl]: Just trying to have them look the same, not trying to torture you

**Pull request merged**